assets/src/components/cluster/queries.js

```diff
@@ -221,3 +221,113 @@ export const CONFIGURATION_OVERLAYS = gql`
   }
   ${ConfigurationOverlayFragment}
 `
+
+export const GLOBAL_SERVICES_QUERY = gql`
```

Comment:
could you use the typed queries for these?  Examples should be in `assets/src/graph` and when you run `yarn graphql:codegen` you'll suddenly get fully typed hooks like `useGlobalServicesQuery` that make typescripting this much simpler